projects/coordinator/src/main/java/org/batfish/coordinator/ExceptionMapper.java

```diff
@@ -0,0 +1,22 @@
+package org.batfish.coordinator;
+
+import com.google.common.base.Throwables;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class ExceptionMapper implements javax.ws.rs.ext.ExceptionMapper<Exception> {
+  @Override
+  public Response toResponse(Exception exception) {
+    if (exception instanceof WebApplicationException) {
+      // An exception that already maps directly to a standard HTTP Response, likely thrown
+      // somewhere inside Jersey aka if the user typed a bad URI.
+      return ((WebApplicationException) exception).getResponse();
```

Comment:
I disagree – BatfishException is used for absolutely everything, E.g., error opening a file, which may well be a backend error. We need to rethink our exception handling if we're going to be able to distinguish user and backend errors. For now, 500 is a safe bet.